src/Data/Fin/Base.agda

```diff
@@ -68,16 +67,15 @@ fromℕ (suc n) = suc (fromℕ n)
 
 -- fromℕ< {m} _ = "m".
 
-fromℕ< : m ℕ.< n → Fin n
-fromℕ< {zero}  {suc n} z<s = zero
-fromℕ< {suc m} {suc n} (s<s m<n) = suc (fromℕ< m<n)
+fromℕ< : ∀ {m n} → .(m ℕ.< n) → Fin n
```

Comment:
I agree with making the argument irrelevant. However is there a reason why we have re-introduced the implicit arguments that are already variables?

Reply:
Hmm, good question. Legacy I think, in terms of restoring the module to its state pre my original investogtaions. But should tidy this up!

Reply:
Done.